managed-databases/postgresql-and-mysql/how-to/manage-snapshots.mdx

```diff
@@ -7,27 +7,26 @@ content:
   paragraph: This page explains how to manage database snapshots
 tags: managed-database database database-snapshot
 dates:
-  validation: 2023-01-19
+  validation: 2023-04-18
   posted: 2021-03-10
 categories:
   - managed-databases
   - postgresql-and-mysql
 ---
 
-[Snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a feature with which you can back up an entire Instance, instead of creating individual automated database backups. It is only available for Database Instances that use Block Storage volumes. 
+[Snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a feature with which you can back up an entire Instance, instead of creating individual automated database backups. It is only available for Database Instances that use Block Storage volumes.
 
-When you set up [autobackups](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup/) for Database Instances that use Block Storage, the automatic backups are created in Snapshot format. 
+When you set up [autobackups](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup/) for Database Instances that use Block Storage, the automatic backups are created in Snapshot format.
```

Comment:
the problem with adding this info is I think It'd become confusing for the client, since even though they're different, they're both named "autobackup". Either way, I think it's more pertinent to include this info in the autobackups doc. I've added this message:

<Message type="important">
  Although they bear the same name, automated snapshots and automated bakcups are different in their design. If you have a Database Instance using Block Storage, it is only possible to use automated snapshots as autobackup.
</Message>

is this ok?

Reply:
Yes, it's good like that, thanks